test/parallel/test-cluster-disconnect-handles.js

```diff
@@ -55,11 +56,30 @@ if (cluster.isMaster) {
     }));
   }));
   process.on('exit', () => assert.deepStrictEqual(handles, {}));
+  process.on('uncaughtException', function(ex) {
+    // Make sure we clean up so as not to leave a stray worker process running
+    // if we encounter a connection or other error
+    if (!worker.isDead()) {
+      if (!isKilling) {
+        isKilling = true;
+        worker.once('exit', function() {
+          throw ex;
+        });
+        worker.process.kill();
+      }
+      return;
+    }
+    throw ex;
+  });
 } else {
   const server = net.createServer(socket => socket.pipe(socket));
-  server.listen(() => {
+  const cb = () => {
     process.send(['listening', server.address()]);
     debugger;
-  });
+  };
+  if (common.hasIPv6)
```

Comment:
I don't know, but this was the only one I personally saw that was causing this problem.